04/app.js

```diff
@@ -2,8 +2,26 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
-        month: '04',
+        day: '23',
+        month: '05',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+
+const currentDay = new Date().getDate() > 9 ? new Date().getDate() : '0' + new Date().getDate();
+let currentMonth = new Date().getMonth();
+currentMonth = parseInt(currentMonth+1) > 9 ? parseInt(currentMonth + 1) : '0' + parseInt(currentMonth+1);
```

Comment:
Przydałoby się napisać funkcję, która tą czynność wykona.
Zobacz, że praktycznie wykonujesz to samo.
Do liczby mniejszej niż 10 dopisujesz zero.
Tutaj kłania się nam zasada DRY -> https://pl.wikipedia.org/wiki/DRY

Reply:
Ok, w następnych postaram się już nie robić tego błędu